src/config.js

```diff
@@ -22,7 +22,12 @@ module.exports = {
     bucket: process.env.AWS_BUCKET,
     path: process.env.AWS_BUCKET_PATH,
     // HTTP cache time in seconds. Defaults to 0.
-    cache: process.env.AWS_CACHE_TIME
+    cache: process.env.AWS_CACHE_TIME,
+    endpoint: process.env.AWS_S3_ENDPOINT,
+    accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+    secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
+    s3ForcePathStyle: process.env.AWS_S3_FORCE_STYLE_PATH || "false",
+    signatureVersion: process.env.AWS_SIGNATURE_VERSION || "v4"
```

Comment:
I don't know that we want to specify defaults here. It'd be nice to let them remain undefined.